ValveResourceFormat/Resource/ResourceTypes/Sound.cs

```diff
@@ -179,12 +211,51 @@ public override void Read(BinaryReader reader, Resource resource)
 
             // v2 and v3 are the same?
 
-            if (resource.Version < 4)
+            if (resource.Version >= 4)
+            {
+                LoopEnd = reader.ReadInt32();
+            }
+
+            ReadPhonemeStream(reader, sentenceOffset);
+        }
+
+        private void ReadPhonemeStream(BinaryReader reader, long sentenceOffset)
+        {
+            if (sentenceOffset == 0)
+                return;
+            
+            Reader.BaseStream.Position = sentenceOffset;
+            
+            var numPhonemeTags = reader.ReadInt32();
+
+            var a = reader.ReadInt32(); // numEmphasisSamples ?
+            var b = Reader.ReadInt32(); // Sentence.ShouldVoiceDuck ?
+
+            // Skip sounds that have these
+            if (a != 0 || b != 0)
             {
                 return;
             }
 
-            LoopEnd = reader.ReadInt32();
+            Sentence = new Sentence();
+
+            Sentence.RunTimePhonemes = new PhonemeTag[numPhonemeTags];
+
+            for (var i = 0; i < numPhonemeTags; i++)
+            {
+                var startTime = reader.ReadSingle();
+                var endTime = reader.ReadSingle();
+                var phonemeCode = (UInt16)reader.ReadInt32();
```

Comment:
Why?

Reply:
Valve aligns it to 4 bytes. Should I really ReadUInt16 and skip 2?

Reply:
If it's alignment bytes, skip them separately.